config/notifies.go

```diff
@@ -280,3 +286,27 @@ func (c *WebhookConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	}
 	return checkOverflow(c.XXX, "slack config")
 }
+
+// OpsGenieConfig configures notifications via OpsGenie.
+type OpsGenieConfig struct {
+	APIKey      string            `yaml:"api_key"`
+	APIHost     string            `yaml:"api_host"`
+	Description string            `yaml:"description"`
+	Details     map[string]string `yaml:"details"`
+
+	// Catches all undefined fields and must be empty after parsing.
+	XXX map[string]interface{} `yaml:",inline"`
+}
+
+// UnmarshalYAML implements the yaml.Unmarshaler interface.
+func (c *OpsGenieConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	*c = DefaultOpsGenieConfig
+	type plain OpsGenieConfig
+	if err := unmarshal((*plain)(c)); err != nil {
+		return err
+	}
+	if c.APIKey == "" {
+		return fmt.Errorf("missing service key in OpsGenie config")
```

Comment:
s/service/API/

Might have been forgotten in other places too after renaming to APIKey.